sunpy/util/cond_dispatch.py

```diff
@@ -78,7 +78,7 @@
 
 from itertools import chain, repeat
 
-from sunpy.extern.six.moves import zip as izip
+from sunpy.extern.six.moves import zip
```

Comment:
also it's ugly having it everywhere in the code, at least the ugly is only in one place in the imports.